src/main/java/MockClass.java

```diff
@@ -1,8 +1,25 @@
 
 public class MockClass {
 
+    /**
+     * cioc
+     * @param seed
+     * @return
+     */
     public int doSomething(int seed) {
 
         return seed + 5;
     }
+
+    public static void main(String args[]) {
+        try {
+            doNothing();
+        } catch (Throwable t) {
+            System.exit(1);  // Default exit code, 0, indicates success. Non-zero value means failure.
+        }
+    }
+
+    private static void doNothing() {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.tiolktrack.byznass.com:9000/coding_rules#rule_key=squid%3AS1186)